### PR TITLE
chore(doc): minor corrections in CONTRIBUTING

### PR DESCRIPTION
**Pull request by lemeurherve**
Added goreleaser requirement and removed outdated output, plus minor corrections.

### CODE REVIEW

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -6,45 +6,32 @@ Thanks for your interest in this project, feel free to ask any questions you may
...
-go build \
-	-ldflags "-w -s \
-        -X \"github.com/updatecli/updatecli/pkg/version.BuildTime=`date -R`\" \
-        -X \"github.com/updatecli/updatecli/pkg/version.GoVersion=go version go1.16.2 linux/amd64\" \
-        -X \"github.com/updatecli/updatecli/pkg/version.Version=v0.0.15-6-g1e24f1d\""\
-        -o bin/updatecli
-```
-
-or using docker
+To build the project, just ensure to have access to the correct golang version and link:https://goreleaser.com/install/[goreleaser installed], then just run `make build`, or use Docker:
```
Comment (dduportal) [inline CONTRIBUTING.adoc:9]:
```suggestion
To build the project, make sure that you link:golang[https://go.dev/] installed, link:https://www.gnu.org/software/make/[GNU Make] and link:https://goreleaser.com/install/[goreleaser installed], then execute the command `make build`, or use Docker with:
```
- Remove occurences of `just` (even though the litteral meaning is "only", it sometimes have a deameaning sense for beginners.
- Add reference to GNU make
-  Add link to golang website

File: CONTRIBUTING.adoc [hunk 2]
```diff
@@ -6,45 +6,32 @@ Thanks for your interest in this project, feel free to ask any questions you may
...
 
 ```
 docker run --rm -v "$PWD":/usr/src/updatecli -w /usr/src/updatecli -e GOOS=windows -e GOARCH=386 golang:1.16 go build -v
 ```
 
 == CONTRIBUTE
 
-They are multiple ways to contribute which don't necessarily involve coding like providing feedback, improving documentation, processes.
-Here I'll just highlight some of them
+They are multiple ways to contribute which don't necessarily involve coding, like providing feedback, improving documentation, processes.
```
Comment (dduportal) [inline CONTRIBUTING.adoc:17]:
```suggestion
They are multiple ways to contribute which don't necessarily involve coding, like providing feedback, improving documentation and processes.
```

**Review by dduportal — COMMENTED** [review]
Thanks for this improvement! I've let minor comments, and i'll let @olblak approve (and merge) if it is ok

**Review by olblak — APPROVED** [review]
Thank you for this improvement.

**Pull request merged**